### ISSUE TITLE
[core] CHRONVER validation seems to be too strict

### ISSUE THREAD

**Issue by Framstag** [jreleaser/jreleaser#1295]
### Task List

- [x] Steps to reproduce provided
- [ ] Stacktrace (if present) provided
- [x] Example that reproduces the problem (link to git repository is ideal)
- [x] Full description of the issue provided (see below)

### Steps to Reproduce

Create a project with CHRONVER and try to release with version 2023.03.25-latest. See https://github.com/Framstag/libosmscout/actions/runs/4519655783/jobs/7960265311

### Expected Behaviour

IMHO 2023.03.25-latest is a allowed chrover version, see https://www.npmjs.com/package/@chronver/regex

### Actual Behaviour

JReleaser claims, that it is not a valid version

This is a simple test with the regular exeption and a copy of the relevant code in JRealeaser itself.

```
tim@kiste:projects/acousticrules ‹main*›$ $JAVA_HOME/bin/jshell
|  Welcome to JShell -- Version 17.0.6
|  For an introduction type: /help intro

jshell> Pattern VERSION_PATTERN = Pattern.compile("^([2-9]\\d{3})\\.(0[1-9]|1[0-2])\\.(0[1-9]|[1-2]\\d|3[0-1])(?:\\.((?:[1-9]\\d*)(?:(?:-[a-zA-Z0-9]+)+(?:\\.[1-9]\\d*)?)?))?$");
VERSION_PATTERN ==> ^([2-9]\d{3})\.(0[1-9]|1[0-2])\.(0[1-9]|[1-2]\d|3 ... 9]+)+(?:\.[1-9]\d*)?)?))?$

jshell> Matcher m = VERSION_PATTERN.matcher("2032.03.26-latest");
m ==> java.util.regex.Matcher[pattern=^([2-9]\d{3})\.(0 ... ?$ region=0,17 lastmatch=]

jshell> System.out.println(m.matches());
false

jshell> Matcher m = VERSION_PATTERN.matcher("2023.03.26-latest");
m ==> java.util.regex.Matcher[pattern=^([2-9]\d{3})\.(0 ... ?$ region=0,17 lastmatch=]

jshell> System.out.println(m.matches());
false

jshell> Matcher m = VERSION_PATTERN.matcher("2023.03.26.1-latest");
m ==> java.util.regex.Matcher[pattern=^([2-9]\d{3})\.(0 ... ?$ region=0,19 lastmatch=]

jshell> System.out.println(m.matches());
true
```

It seems like a version most always have four parts, three parts are not allowed.

**Comment by aalmiray** [comment]
Test `version_parsing` @  https://github.com/jreleaser/jreleaser/blob/main/api/jreleaser-model-api/src/test/java/org/jreleaser/version/ChronVerTest.java is missing a case that matches "yyyy.mm.dd-tag"

Regular expression @ https://github.com/jreleaser/jreleaser/blob/10742852a58603b852c6debafb9d50034d956b58/api/jreleaser-model-api/src/main/java/org/jreleaser/version/ChronVer.java#L37 

should be updated

**Comment by aalmiray** [comment]
🎉 This issue has been resolved in `v1.6.0` ([Release Notes](https://github.com/jreleaser/jreleaser/releases/tag/v1.6.0))
